blocks/v2-columns/v2-columns.css

```diff
@@ -18,13 +22,25 @@
 }
```

Comment:
can you remove this max-width just for this variant to be applied from min-width: 744px. it looks odd from 512px to 743px IMHO